smac/utils/io/traj_logging.py

```diff
@@ -172,6 +176,81 @@ def _add_in_aclib_format(self, train_perf: float, incumbent_id: int,
             json.dump(traj_entry, fp)
             fp.write("\n")
 
+    def _add_in_alljson_format(self, train_perf: float, incumbent_id: int,
+                               incumbent: Configuration, ta_time_used: float,
+                               wallclock_time: float):
+        """Adds entries to AClib2-like (but with configs as json) trajectory file
+
+        Parameters
+        ----------
+        train_perf: float
+            Estimated performance on training (sub)set
+        incumbent_id: int
+            Id of incumbent
+        incumbent: Configuration()
+            Current incumbent configuration
+        ta_time_used: float
+            CPU time used by the target algorithm
+        wallclock_time: float
+            Wallclock time used so far
+        """
+        # Silent support for passing configuration-dict's directly:
+        inc_dict = incumbent
+        if isinstance(incumbent, Configuration):
+            inc_dict = incumbent.get_dictionary()
+
+        traj_entry = {"cpu_time": ta_time_used,
+                      "total_cpu_time": None,  # TODO: fix this
```

Comment:
I don't know whether we will fix/support that anyway in the future. Therefore, we should drop that here.

Reply:
Fair enough. I will remove all TODO-tags in the file.